src/run_sign.py

```diff
@@ -15,7 +15,7 @@
 from sign_workflow.signer import Signer
 from system import console
 
-ACCEPTED_SIGNATURE_FILE_TYPES = [".sig"]
+ACCEPTED_SIGNATURE_FILE_TYPES = [".sig", ".asc"]
```

Comment:
How will this work? The signer client does not support file types yet. `sig` is what it returns by default. If we add this, either the signer client needs to support this or we need to add additional code on our end to convert `sig` files to `.asc` files

Reply:
I agree with @gaiksaya that we need to 1st let signer client to have this function, before adding it to the signing workflow.

Reply:
AFAIK It does support this functionality, if you look further down the default is `.asc` if no type is supplied

Reply:
If this is not correct and we cannot use `.asc` then that is another issue entirely - and yes then I should just close this PR until that is resolved

Reply:
I thought the default is to binary sig file?
It does not have a enarmor function to change to ascii?
Please correct me if not the case.

Reply:
Here is the related issue https://github.com/opensearch-project/opensearch-signer-client/issues/10

Reply:
Default is `.asc` here https://github.com/opensearch-project/opensearch-build/blob/c18e1363910031d0368aa4a949f6dc507c442d72/src/run_sign.py#L26 where is technically not an enarmor output

Reply:
Out of curiosity, If signer client doesn't have functionality to sign `.asc`, how did we do signing files for maven release [here](https://github.com/opensearch-project/opensearch-infra/blob/f71d0d72f79c02dcd7cd00c7aabadc0289e12d12/jenkins/jobs/OpenSearch_CI/release_ci/opensearch_maven_release/Jenkinsfile#L58)? Wers this output fake `.asc` files without actually converted?

Reply:
The reason we left out the `.asc` here was because the `.asc` files created by the signer client are technically incorrect as @peternied pointed it out in the prev PR - 

https://github.com/opensearch-project/opensearch-build/pull/1352#discussion_r771459100

I think a better solution would be to make sure the params pick out default value in case a `null`, `None` or an empty string is passed

Reply:
We've been working around opensearch-project/opensearch-signer-client#10 since OpenSearch 1.0.0  Lets just implement the work around in this code base as we aren't getting traction on that issue.